### PR TITLE
Fix JSON array (fix issue #13)

### PR DESCRIPTION
**Pull request by lksv**
...solution is based on @sarge's comment, therefore credits belong to him:)

### CODE REVIEW

File: lib/rack/parser.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class Parser
     POST_BODY  = 'rack.input'.freeze
     FORM_INPUT = 'rack.request.form_input'.freeze
     FORM_HASH  = 'rack.request.form_hash'.freeze
+    PARSEER_RESULT  = 'rack.parser.result'.freeze
```
Comment (sarge) [inline lib/rack/parser.rb]: Typo I think, PARSER_RESULT is the spelling you want:)

Comment (lksv) [inline lib/rack/parser.rb]: Thanks. 
Fixed  (commit amended).

**Comment by sarge** [comment]
:+1: Looks good

**Comment by achiu** [comment]
I've merged this PR and released a new version on rubygems https://github.com/achiu/rack-parser/releases/tag/v0.7.0

**Pull request merged**